frontend/src/app/settings/core/announcement-settings/edit-announcement/edit-announcement/edit-announcement.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<form [formGroup]="announcementForm" class="collab-card">
+  <mat-form-field class="w-full" appearance="fill">
```

Comment:
```suggestion
  <mat-form-field class="w-full" appearance="fill" subscriptSizing="dynamic">
```

If you add `subscriptSizing="dynamic"` to all form fields, the space for potential messages is added on demand. This can be used to reduce the space between the form fields a bit.